Update lifters.csv

TrystanOakley opened the pull request:
Changed Gaylon Black to Perseus Black per his request. He officially changed it on his membership card with the federation as well so. @sstangl

sstangl commented:
Needs an entry in the name change database also --

https://github.com/sstangl/openpowerlifting/blob/master/docs/lifter-name-corrections.md

Also uspa/0858.

TrystanOakley commented:
I changed it in both meets, would you still like me to add it to he name corrections since I changed it in all of his meets and he said he will be using that name from now on?

TrystanOakley commented:
Okay I’ll add it. He’s only done the two meets though. I verified it with
him.

On Tue, Nov 14, 2017 at 10:56 AM Sean Stangl <notifications@github.com>
wrote:

> I only see it changed in one meet, but yeah, please add to the
> name-corrections.dat file -- it's possible that he has some meets that we
> haven't entered yet, and this lets us forget about it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_sstangl_openpowerlifting_pull_365-23issuecomment-2D344324315&d=DwMFaQ&c=7ypwAowFJ8v-mw8AB-SdSueVQgSDL4HiiSaLK01W8HA&r=QrLkh9Hgm4OFYnkpWoWTFOD9LXMNtJPJfeRAiZl_Igk&m=8w4Hp_wt3wP1UeqdrbUE3-n260Rz5hiLWJoZvvSl1zA&s=ckL9dmeI1_EO33Co322EM0cGdJ6jEwIaSABdMqptuzU&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_Ac52pJdQttHL3tjtYq528RqPGhT9SrONks5s2cYhgaJpZM4Qdj6u&d=DwMFaQ&c=7ypwAowFJ8v-mw8AB-SdSueVQgSDL4HiiSaLK01W8HA&r=QrLkh9Hgm4OFYnkpWoWTFOD9LXMNtJPJfeRAiZl_Igk&m=8w4Hp_wt3wP1UeqdrbUE3-n260Rz5hiLWJoZvvSl1zA&s=zjOZI6uLk1pj3pQWGl4_7x2vijlyxaaXQ5hPUNa_X8E&e=>
> .
>
